src/05-objects.js

```diff
@@ -118,7 +142,12 @@ function addKeyToObject(object, key) {}
  *  addKeyValuePairToObject({ up: true }, "left", false);
  *  //> { up: true, left: false }
  */
-function addKeyValuePairToObject(object, key, val) {}
+function addKeyValuePairToObject(object, key, val) {
+  // console.log(object,key,val)
```

Comment:
Lets also keep an eye out for our console.log statements.  These won't break our code but we should always remove commented out code or console.log() statements before we push.